### ISSUE TITLE
Need support for gated SAE

### ISSUE THREAD

**Issue by yangjingyuan** [callummcdougall/sae_vis#54]
AssertionError: If encoder isn't an AutoEncoder, it should have weights 'W_enc', 'W_dec', 'b_enc', 'b_dec'

Gated SAE do not have b_enc and it seems AutoEncoder is not suitable for gated SAE.

**Comment by callummcdougall** [comment]
Noting that there's a fork of this repo which is used for sae-lens, and if most people will be working with sae-lens going forwards, then I think the probability of building in support for gated SAEs here is somewhat low. I'd be updated if there are a lot of people using sae-vis but not sae-lens though (although caveat that I'm mostly too busy to keep contributing to this library in major ways, so probably someone else would be implementing gated SAEs if it does end up happening here)

**Comment by callummcdougall** [comment]
Makes sense, although I would guess there's also an advantage to supporting general non-SAE-lens architectures? Unless it's easy enough to wrap everything in an SAE from sae-lens that it's fine to assume it's sae-lens type?

**Comment by jbloomAus** [comment]
https://github.com/jbloomAus/SAEDashboard supports any SAE Lens SAE.
Planning to promote more widely shortly.

**Comment by yangjingyuan** [comment]
I see. It's better to have a united mechanical interpretability tool(combine transformer-lens, sae-lens, sae-vis,...) that can do circuit discovery, SAE, visualization, activation steering, etc., which can use multiple nodes for multiple GPUs for large LLMs.
